modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -46,29 +76,34 @@ If this step is not complete, finalizers make it difficult to fully uninstall th
 +
 [source,terminal]
 ----
-$ oc delete customresourcedefinitions.apiextensions.k8s.io workspaceroutings.controller.devfile.io
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceroutings.controller.devfile.io
 ----
 +
 [source,terminal]
 ----
-$ oc delete customresourcedefinitions.apiextensions.k8s.io components.controller.devfile.io
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
 ----
 +
+. Remove the `DevWorkspace-Webhook-Server` deployment and mutating and validating webhooks:
```

Comment:
s/the `DevWorkspace-Webhook-Server` deployment and mutating and validating.../the `DevWorkspace-Webhook-Server` deployment, mutating, and validating...